### DISCUSSION TITLE
Application Rejection Issue for Online Course Verification

### DISCUSSION THREAD

**Discussion by Alex-Dislva** [community/community#63214] [GitHub Education]
### Hi everyone,

Hello everyone,

I hope you're all doing well. I wanted to bring up an issue I've been facing regarding the verification of my online course on GitHub Education. I am a verified IDE student at Manipal Online, and my student ID card clearly states that I am enrolled in an online course. However, despite my best efforts, my application for GitHub Education benefits keeps getting declined repeatedly.

I have followed all the necessary steps and provided accurate documentation to prove my enrolment status, but the rejections continue to occur. This has been quite frustrating for me as I believe I meet all the requirements for the benefits, and I'm unable to access the advantages that come with the GitHub Education package.

I wanted to know if anyone else is facing similar issues or has encountered this problem in the past. If you have any insights or suggestions on how to resolve this matter, I would greatly appreciate your input. Perhaps some of you have successfully navigated through a similar situation and can offer guidance on what steps to take next.

**Discussion Points:**

**Documentation Accuracy**: Have you ensured that the documents you provided for verification are clear, accurate, and up-to-date? Sometimes, small discrepancies in the documentation can lead to rejections.

**Communication with GitHub Support:** Have you reached out to GitHub Education support regarding this issue? They might be able to provide more specific information about why your application is being declined and suggest potential solutions.

**Verification Process:** Has anyone experienced a particularly smooth verification process? It would be helpful to hear from those who have successfully verified their online course enrolment and how they went about it.

**Time Frame:** How long have you been experiencing these rejections? It's possible that there might be delays in the verification process due to high demand or other factors.

**Alternative Documents:** Are there any alternative documents you can provide to support your enrolment status? For instance, some universities provide enrolment letters or official statements that could serve as supplementary evidence.

**Double-Checking Requirements:** It might be worth revisiting the GitHub Education requirements to ensure you haven't missed any critical steps or criteria.

**Community Support:** Let's collaborate to help each other out. If you've successfully resolved a similar issue or have any suggestions, please share your experiences and advice.

**Patience and Persistence:** Dealing with administrative matters can be frustrating, but it's important to stay patient and persistent. Share your experiences of how you managed to overcome obstacles in the past.

Please feel free to share your thoughts, experiences, and suggestions on this matter. Let's work together to find a solution and ensure that all eligible students can access the benefits they deserve through GitHub Education.

Best regards,
Alex D'Silva

**Comment by humairshoukat** [comment]
try submitting your student ID card pic fro your phone. try taking the student card pic while verifying.
